examples/basic_vpc_peering/versions.tf

```diff
@@ -16,4 +16,13 @@
 
 terraform {
   required_version = ">=0.12.6"
+
+  required_providers {
+    google = {
+      version = "~> 4.0"
```

Comment:
Ah, I think I was unclear. For the examples where the module is pinned, we should stick to `~> 3.x`.

Reply:
Makes sense, so the live examples still work... those updates were made before I was clear on that. I wonder if we'll still end up with a version clash, but I'll make that correction now and let's see.

Reply:
Done.

Reply:
Looks like linters are still failing. Please fix.

Reply:
Isn't that an expected conflict between the versions required by the pinned and non-pinned versions?

I need to head out for the evening now, will take a further look tomorrow.

Reply:
Nope, you'll notice it is failing on just `test/setup` now. Which you can actually leave completely untouched.

Reply:
Great, that did it, thanks. Actually not _completely_ untouched, as it was also using the old form of representing constraints.